webapp/src/Controller/Jury/InternalErrorController.php

```diff
@@ -161,9 +166,21 @@ public function handleAction(int $errorId, string $action): RedirectResponse
                     $internalError->getContest(),
                     true
                 );
-                $this->dj->auditlog('internal_error', $internalError->getErrorid(),
-                                    sprintf('internal error: %s', $status));
+                $affectedJudgings = $internalError->getAffectedJudgings();
+                if ($affectedJudgings !== null) {
+                    $skipped = array();
+                    $this->rejudgingService->createRejudging(
+                        'Internal Error ' . $internalError->getErrorid() . ' resolved',
+                        JudgeTask::PRIORITY_DEFAULT,
```

Comment:
You create new tasks right? So what happens when on t1 an executable breaks,
t2,t3 we get a lot of tasks so we have a backlog
t4 we fix the internal error,

would we first solve the tasks for t1 (created at t4) or the ones for t2?

Reply:
By default judging fairness right now, i.e. teams that didn't occupy a judgehost for the longest time come first.

But we can make this configurable in a separate PR.